diesel/Cargo.toml

```diff
@@ -33,6 +33,9 @@ bigdecimal = { version = ">=0.0.13, < 0.4.0", optional = true }
 bitflags = { version = "1.2.0", optional = true }
 r2d2 = { version = ">= 0.8.2, < 0.9.0", optional = true }
 itoa = { version = "1.0.0", optional = true }
+# newer versions of `time` have a MSRV of 1.57.0, which is newer than
+# diesel's own MSRV of 1.56.0
+time = { version = "=0.3.9", optional = true, features = ["macros", "formatting", "parsing"] }
```

Comment:
It is fine to also allow newer time versions here, as we define our MSRV not by using the latest variant of all dependencies, but the oldest one. Or better: There must be a dependency version that builds fine with the given MSRV, we do not require that to be true for all dependency versions.

Reply:
Additionally we want to include `time` in the `extras` feature in line 52